Torantulino/Auto-GPT
# Fix the debug mode flag to set it in the config object and check properly

### Background

Running the program with  --debug option to enable the Debug Mode does not work.

### Changes

1. Set the debug mode in the config object properly
2. Check the debug mode from the config object properly in the if conditions.

### Test Plan

1. Run the program with --debug parameter from the command line
2. Verify the correct debug messages in console

### Change Safety

- [*] I have considered potential risks and mitigations for my changes

It's a simple change to make the debug mode work. It does not need a recession test case.
